src/core/utils/fs.cpp

```diff
@@ -453,6 +453,18 @@ QString Utils::Fs::QDesktopServicesDownloadLocation()
 #endif
 #endif
 
+#if defined(Q_OS_WIN)
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
+    if (QSysInfo::windowsVersion() <= QSysInfo::WV_XP)  // Windows XP
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::DocumentsLocation)).absoluteFilePath(
+                    QCoreApplication::translate("fsutils", "Downloads"));
+    else
+        return QDir(QDesktopServices::storageLocation(QDesktopServices::HomeLocation)).absoluteFilePath("Downloads");
+#else
+    return QStandardPaths::writableLocation(QStandardPaths::DownloadLocation);
```

Comment:
This potentially could be used for Linux too. I don't know if Qt5 respects `XDG_DOWNLOAD_DIR` though. This isn't high prio, since the other code "just works".
However, this presents an issue with qt5 and windows XP. It points to `C:\Documents and Settings\<username>\My Documents` instead of `C:\Documents and Settings\<username>\My Documents\Downloads` like we do now.
Why? Well, Windows XP doesn't actually have a downloads folder. It is a de fact standatd to put them to "Downloads" inside "My Documents".

Btw, do you have a non-english Windows Vista+ to test? I think the "Downloads" gets translated in other languages. So does this function still point to the correct folder?